### PR TITLE
Updated translations from Phrase

### PR DESCRIPTION
**Pull request by terraware-phrase**
Exported the most recent locale files from Phrase

### CODE REVIEW

File: src/main/resources/i18n/Enums_es.properties [hunk 1]
```diff
@@ -19,6 +19,18 @@ accelerator.DeliverableCategory.SupplementalFiles=Archivos suplementarios
...
 # Noun, as in a legal document
 accelerator.DeliverableType.Document=Documento
+accelerator.ScoreCategory.Carbon=Carbono
+accelerator.ScoreCategory.ClimateImpact=Impacto climático
+accelerator.ScoreCategory.Community=Comunidad
+accelerator.ScoreCategory.ExpansionPotential=Capacidad de ampliación
+accelerator.ScoreCategory.ExperienceAndUnderstanding=Experiencia y percepción
+accelerator.ScoreCategory.Finance=Finanzas
+accelerator.ScoreCategory.Forestry=Silvicultura
+accelerator.ScoreCategory.GIS=Sistema de información geográfica (GIS)
```
Comment (tommylau523) [inline src/main/resources/i18n/Enums_es.properties]: @alexTerraformation It seems like in other places, GIS should not be translated. What should I do here?

Comment (sgrimm) [reply src/main/resources/i18n/Enums_es.properties]: I think this is one of those cases where we should just override the translation. I'll go update this in the Phrase UI and tell it to update the PR.

Comment (tommylau523) [reply src/main/resources/i18n/Enums_es.properties]: Thanks! In a separate UI I have also marked these an not localizable. Perhaps we can just close this PR.

Comment (sgrimm) [reply src/main/resources/i18n/Enums_es.properties]: Also, the translator got it wrong here -- the Spanish translation for "GIS" is actually "SIG" (it's correct in other strings).

**Pull request merged**